composer.json

```diff
@@ -15,13 +15,28 @@
             "email": "damian@silverstripe.com"
         }
     ],
-    "minimum-stability": "dev",
     "require": {
-        "silverstripe/framework": "~3.1"
+        "silverstripe/recipe-cms": "^1"
```

Comment:
This change will introduce a bunch of new dependencies that weren't originally there - I suggest we stick with `"silverstripe/framework": "^4.0"` instead

Reply:
Ah yeah, I didn't make it clear... recipes should never be a dependency of anything, except other recipes.

It can be done (inlined recipes become `provides: [ 'therecipe' ]` but it's not a direct dependency.